css/style.css

```diff
@@ -0,0 +1,97 @@
+@media only screen and (max-width: 600px) {
+  body {
+    background-color: lightblue;
+    display: flex;
+  }
+}
+
+#navbar {
+  width: 100%;
+  background-color: brown;
+  position: fixed;
+  top: 0;
+}
+
+#navbar a {
+  float: right;
+  margin: 0 15px;
+  padding-bottom: 15px;
```

Comment:
I suggest centering items on the navigation bar.

```suggestion
  You can put    `padding: 15px;` instead of `padding-bottom: 15px;` 
```
![Screenshot from 2021-12-06 16-28-00](https://user-images.githubusercontent.com/62058680/144874612-58ec5cef-0162-4e28-a310-416f351a5b3d.png)

Reply:
- [ ] I think it would be better to explain your suggestions by using GitHub inline comments feature instead of directly writing on the style.css file. Because this may confuse the student.

Reply:
I don't understand I have written nothing on the .css file, this is the GitHub comment !!!